scripts/release/README.md

```diff
@@ -0,0 +1,18 @@
+# Enceladus Release Scripts
+
+## release_prepare
+
+`release_prepare.sh` / `release_prepare.cmd`
+
+* Merges the current _develop_ branch into the _master_ branch, while removing the `_SNAPSHOT` suffix from the modules' version.
+* In the _develop_ branch it increases the minor version of all the modules of the project.
+* It commits and pushes the changes.
```

Comment:
I believe that the final tag version is also created and pushed in the process, which is significant and should be mentioned here, too, for completeness.